quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/MethodOverloadTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Quasar: lightweight threads and actors for the JVM.
+ * Copyright (c) 2015-2017, Parallel Universe Software Co. All rights reserved.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ *
+ *   or (per the licensee's choosing)
+ *
+ * under the terms of the GNU Lesser General Public License version 3.0
+ * as published by the Free Software Foundation.
+ */
+package co.paralleluniverse.kotlin.fibers.lang
+
+import co.paralleluniverse.common.util.SystemProperties
+import co.paralleluniverse.fibers.Fiber
+import co.paralleluniverse.fibers.Suspendable
+import org.junit.Assume
+import org.junit.Test
+import kotlin.test.assertNotNull
+
+class MethodOverloadTest {
+
+    @Suspendable
+    fun function() {
```

Comment:
Would you be able to try running this test on Quasar 0.8.x (JDK11 only version) and JDK11 (they're all OpenJDK derived, but Azul as an example) ?

Reply:
Yes. I can do that.

Reply:
I've ran the test and found that this bug is not present in Java 11. It appears to produce different byte code. I've added more details to the jira ticket.